src/components/HomePage/House/House.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, Button, Checkbox } from '@mui/material';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import noPhoto from '../../../assets/images/noPhoto.png';
+import styles from './House.module.scss';
```

Comment:
The order of import could use prettier to remove the warnings.

Reply:
I use prettier but I don't get any warnings, and I didn't disable anything - so that's strange